src/main/java/org/nazar/service/EmailService.java

```diff
@@ -0,0 +1,42 @@
+package org.nazar.service;
+
+import org.nazar.service.smtp.SmtpAuthenticator;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+
+public class EmailService {
+    private final Authenticator authenticator = new SmtpAuthenticator();
+
+    public static final String TO = "nazarvlk793@gmail.com";
+    public static final String FROM = "nazar.valko09@gmail.com";
+
+    public static final String PASSWORD = "gxib gftv jryu noyn";
```

Comment:
Is it good idea to share password in code? I think not. Think over how to fix this issue